.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,45 @@
+---
+# Tools that can save round-trips to github and a lot of time:
+#
+# yamllint -f parsable pull_request.yml
+# pip3 install ruamel.yaml.cmd
+# yaml json pull_request.yml
+# yaml merge-expand pull_request.yml exp.yml &&
+#    diff -b -u pull_request.yml exp.yml
+
+name: Pull Requests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - {uses: actions/checkout@v2, with: {fetch-depth: 0}}
+      - name: shellcheck
+        env:
+          BASE_REF: ${{ github.base_ref }}
+        # Triple-dotted 'origin/master...HEAD'
+        run: ./tools/CI/check-gitrange.bash origin/${BASE_REF}...HEAD
+               text/x-shellscript shellcheck -x
+
+  pylint:
+    runs-on: ubuntu-latest
+    steps:
+      # :-( https://github.community/t/support-for-yaml-anchors/16128
+      - {uses: actions/checkout@v2, with: {fetch-depth: 0}}
+      - uses: actions/setup-python@v2
```

Comment:
is this using python2? is that why your import is failing

Reply:
Thanks, you pushed me to take a closer look and I found that `ubuntu-latest` still means 18.04 on github. I changed it to `ubuntu-20.04` (what `.travis.yml` had) and that single known regression is now gone! See latest result in #558 

`latest` wasn't a good idea  anyway, very poor "version control" I'm glad this made me change it. I bet github hasn't moved it because they're too afraid to break too much stuff in one go :-)

Gory details: `pylint` was a python2 script in 18.04 and apparently not capable of analyzing a `#!/usr/bin/python3` script.